JS/ViewControllers/librarySelectMediaLanguageViewController.js

```diff
@@ -0,0 +1,175 @@
+                          <tbody id="bodyTable">
+                            <tr>
+                            </tr>
+                          </tbody>
+                        </table>
+                    </form>
+                </div>
+                <div class="row row-20">
+                    <div class="col s12 center">
+                        <h6><p id="notSelected"></p></h6>
+                        <a id="select-media-button" class="waves-effect waves-light btn-large">select</a>
+                    </div>
+                </div>
+                <div class="row row-20">
+                    <div class="col s12">
+                        <a class="btn-small left" href="media_select_FL.html">Change media language</a>
+                        <a class="btn-small right" href="index.html">Back to my menu</a>
+                    </div>
+                </div>
+            </section>
+        `;
+        super.renderHtml(htmlView);
+      }
+
+      setupProperties() {
+          this.selectMediaButton = $('#select-media-button');
+          this.getAvailLessons();
+          this.setAvailLessons();
+      }
+
+      setupEventListeners() {
+          this.selectMediaButtonClicked = this.selectMediaButtonClicked.bind(this);
+          this.selectMediaButton.on('click', this.selectMediaButtonClicked);
+      }
+
+      presentNextController() {
+          const settingsViewController = new SettingsViewController();
+          settingsViewController.lesson = this.lesson;
+          settingsViewController.language = this.language;
+          this.navigationController.present(settingsViewController);
+      }
+
+      selectMediaButtonClicked(){
+          this.getSelectedLesson();
+          if (this.lesson != "Null"){
+              $("#notSelected").text("");
+              this.presentNextController();
+          }
+          else{
+              $("#notSelected").text("Nevybrala sa ziadna lekcia!");
+          }
+      }
+
+      // Private Methods
+
+      // get available lessons from online library through Ajax request
+      getAvailLessons(){
+        let mbpPrimaryLang = this.shortLanguage;
+        var tmp = "Null";
+            $.ajax({
+                type: 'POST',
+                url: "http://www.st.fmph.uniba.sk/~hrebenar3/Projects/MBP/api.php",
+                data:{
+                    action: "get-lectures-in-lang",
+                    primaryLang: mbpPrimaryLang,
+                },
+                dataType: 'json',
+                async: false,
+                success:function(data){
+                    tmp = data;
+                },
+                error:function (XMLHttpRequest, textStatus, errorThrown) {
+                    console.log("FAIL");
+                }
+            });
+            this.listOfLessons = tmp;
+      }
+
+      // set available lessons from atribute this.listOfLessons
+      setAvailLessons(){
+          var table = document.getElementById("avail-lecture");
```

Comment:
Toto je nejaka table ktora tam je stale ci? Ak ano patri to do `setupProperties()` a nie sa stale do okola na to dopytovat

Reply:
A ano je to tabulka, ktora tam je stale,upravim to a dam to do setupProperties()